test/modules/layers/test_position_embedding.py

```diff
@@ -17,63 +17,80 @@
 class TestBroadcastedPositionEmbedding:
     @pytest.fixture(scope="class")
     def pos_emb(self):
-        return BroadcastedPositionEmbedding(
-            shape=(1, 2),
+        _pos_emb = BroadcastedPositionEmbedding(
```

Comment:
In this case I am trying to avoid confusion with the function name "pos_emb". 
Although one leading underscore is proposed by PEP8 to denote non-public methods and instance variables for a class, it doesn't explicitly discourage borrowing this pattern for other use cases, e.g., people use one leading underscore in `for _x in [1, 2, 3, 4]`.